packages/waggle/src/storage/storage.test.ts

```diff
@@ -2,24 +2,27 @@ import fs from 'fs'
 import path from 'path'
 import PeerId from 'peer-id'
 import { Config } from '../constants'
-import { createLibp2p, createTmpDir, tmpZbayDirPath, rootPermsData } from '../common/testUtils'
+import { createLibp2p, createTmpDir, tmpZbayDirPath, rootPermsData, createMinConnectionManager } from '../common/testUtils'
 import { Storage } from './storage'
 import * as utils from '../common/utils'
 import { createUserCsr, createUserCert, configCrypto } from '@zbayapp/identity'
 import { DirResult } from 'tmp'
+import { ConnectionsManager } from '../libp2p/connectionsManager'
 jest.setTimeout(30_000)
 
 let tmpDir: DirResult
 let tmpAppDataPath: string
 let tmpOrbitDbDir: string
 let tmpIpfsPath: string
+let connectionsManager: ConnectionsManager
 let storage: Storage
 beforeEach(() => {
   jest.clearAllMocks()
   tmpDir = createTmpDir()
   tmpAppDataPath = tmpZbayDirPath(tmpDir.name)
   tmpOrbitDbDir = path.join(tmpAppDataPath, Config.ORBIT_DB_DIR)
   tmpIpfsPath = path.join(tmpAppDataPath, Config.IPFS_REPO_PATH)
+  connectionsManager = createMinConnectionManager({ env: { appDataPath: tmpAppDataPath }, torControlPort: 12345 })
```

Comment:
Is this hardcoded torControlPort safe? Shouldn't we find port dynamically?

Reply:
We should. Done.